### DISCUSSION TITLE
Moving packages

### DISCUSSION THREAD

**Discussion by stephenmathieson** [clibs/clib#231] [General]
I have a large number of small C packages I'm not really maintaining anymore (example [`case.c`](https://github.com/stephenmathieson/case.c), [`mkdirp.c`](https://github.com/stephenmathieson/mkdirp.c), etc). I'm wondering if I should move them into the `clibs` org in order to enable others to work on them (if needed).

Any thoughts/recommendations?

**Comment by jwerle** [comment]
that sounds good to me!

**Reply by stephenmathieson** [reply]
Cool ok. I'll try to get started on this soon. Will probably make a robot do it, as there are too many repos for me to worry about 😬
